Jamroot.jam

```diff
@@ -0,0 +1,20 @@
+path-constant LIBTORRENT_ROOT : . ;
+
+rule asio_src ( properties * )
+{
+	local result ;
+
+	if ! <define>TORRENT_BUILDING_STATIC in $(properties)
```

Comment:
that check is inverted obviously, but if you invert it and return early when linking statically, you don't have to indent the rest of the function